packages/jest-circus/src/utils.js

```diff
@@ -110,46 +110,59 @@ const getEachHooksForTest = (
   return result;
 };
 
+const _makeTimeoutMessage = (timeout, isHook) => {
+  const message = `Exceeded timeout of ${timeout}ms for a ${isHook ? 'hook' : 'test'}.`;
```

Comment:
What do you think about also receiving `testEntry.name` as a parameter? So that it is a bit more explicit about in which test this happened